composer.lock

```diff
@@ -4,8 +4,43 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "ff2c1392af4a32ee1bda12bf7c90a4ce",
+    "content-hash": "df773ab4a17b82868a21f8a1b87d49cf",
     "packages": [
+        {
+            "name": "libero/content-page-bundle",
+            "version": "dev-master",
+            "dist": {
+                "type": "path",
+                "url": "./vendor-extra/ContentPageBundle",
+                "reference": "a334e0f10e447c85dbc6a543dc9d0864a458139a",
```

Comment:
@stephenwf Looked at not having to set the `version` manually, looks like this is a hash of the bundle contents so is enough?

Reply:
oh nice one!